server/text/codap-api-documentation-edited.md

```diff
@@ -419,7 +419,6 @@ The selectionList object is just an array of case ids.
   legendAttributeName: /* {String} An attribute name within the data context */
   enableNumberToggle: /* {Boolean} whether the numberToggle display should be presented in this graph */
   numberToggleLastMode: /* {Boolean} whether numberToggle display should be in "last mode" */
```

Comment:
In some of the JSON examples comments are delimited with `//`, and in others with `/*  ... */`.
Both are non-standard since JSON doesn't actually support comments, but I'd stick to one style to limit possible confusion.